### ISSUE TITLE
Subscription query issue

### ISSUE THREAD

**Issue by meet-logicwind** [Gazuntype/GraphQL-Unity-Client#19]
The subscription query throws error can't connect to the remote server while other queries are working properly. I am following the official tutorial from the following link.

[](https://hasura.io/learn/graphql/unity/matchmaking/2-unity-integration/)

I tried all possible ways including recreating the table, recreating a subscription query in the graph client but nothing worked! Subscription is working fine in the graphql editor but throws an error in Unity. I have attached an editor logs just for reference.

![Screenshot 2020-10-06 195320](https://user-images.githubusercontent.com/69183578/95214524-a02f8d00-080d-11eb-80cd-2439d6a2c768.png)

If you will need a project please let me know I will share it with you.

**Comment by Migueton** [comment]
Hello, I have the same error, the problem is the websocket is not connecting with the server. Any ideas around here to help? I am not familiar with Auth, not sure if it could be that

**Comment by cleverson-schmidt** [comment]
Hi,

I'm getting the same problem. Has anyone found a solution for it?
I got the exception when trying to subscribe to a GraphQl API running on AWS AppSync.
Thanks in advance!

**Comment by Migueton** [comment]
@cleverson-schmidt sadly I didnt find any. I think the developers dont check github at all. I suggest you look for answers in another place.

**Comment by cleverson-schmidt** [comment]
Thank you @Migueton!
Did you find any other GraphQL client for Unity that is reliable? It's really sad that we can't find an oficial library from Amazon.

**Comment by Migueton** [comment]
no, I just tried another solutions rather than GraphQL. Cant remember now honestly
